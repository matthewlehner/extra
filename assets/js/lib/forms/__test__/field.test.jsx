// @flow

import React from "react";
import renderer from "react-test-renderer";
import Field from "../field";

describe("Field component", () => {
  it("renders correctly", () => {
    const tree = renderer.create(
      <Field label="Channel">
        Children are here.
      </Field>
    );

    expect(tree).toMatchSnapshot();
  });
});
