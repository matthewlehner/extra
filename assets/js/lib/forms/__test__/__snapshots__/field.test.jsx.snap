// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Field component renders correctly 1`] = `
<div
  className="form__control-group"
>
  <label
    className="label"
    htmlFor="Channel"
  >
    Channel
  </label>
  Children are here.
</div>
`;
